libs/full/async_mpi/include/hpx/async_mpi/mpi_future.hpp

```diff
@@ -25,8 +27,19 @@
 
 namespace hpx { namespace mpi { namespace experimental {
 
+    // -----------------------------------------------------------------
+    namespace detail {
+
+        using request_callback_function_type = hpx::util::unique_function_nonser<void(int)>;
+
+        HPX_EXPORT void add_request_callback(request_callback_function_type&& f, MPI_Request req);
+        HPX_EXPORT void register_polling(hpx::threads::thread_pool_base&);
+        HPX_EXPORT void unregister_polling(hpx::threads::thread_pool_base&);
+
+    }    // namespace detail
+
     // by convention the title is 7 chars (for alignment)
-    using print_on = debug::enable_print<false>;
+    using print_on = debug::enable_print<true>;
```

Comment:
Just a reminder to disable this before merging.

Reply:
Yes I'll drop my 2 `tmp!` commits before marking that as ready for review